src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -161,6 +162,16 @@ def _parse_tool_from_function(f, initialize_inputs=None, gen_custom_type_conn=Fa
     )
 
 
+def _should_add_index(tool_type):
```

Comment:
please replace with a better function name to explain why should add index?

Reply:
Changed to `should_add_input_index_for_ui_to_keep_order`, thanks.